css/styles.css

```diff
@@ -586,16 +566,125 @@ footer {
   flex-direction: row;
   justify-content: space-around;
   width: 200px;
-  list-style: none;
   margin: 0;
   padding: 0;
   align-items: center;
 }
 
-#hamburguerLink ul {
-  list-style: none;
+/* Details Popup */
+.popupContent {
+  display: none;
+  position: fixed;
+  width: 90%;
+  height: 100%;
+  top: 5%;
+  left: 5%;
+  right: 5%;
+  z-index: 100;
+}
+
+.detailsContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: rgba(255, 255, 255, 0.16);
+  border: 1px solid #fff;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+}
+
+.heading {
+  display: flex;
+  justify-content: space-between;
+  width: 90%;
+  margin-left: 30px;
+}
+
+.close {
+  padding: 8px;
+  position: static;
+  width: 20px;
+  height: 20px;
+  right: 0;
+  top: calc(50% - 40px / 2 - 9px);
+  border-radius: 8px;
+  margin: 15px 15px;
+}
 
-  /* height: 106%; */
+.snapshoot {
+  background: url(../images/SnapshootPortfolio.png) no-repeat;
+  width: 279px;
+  height: 181px;
+}
+
+.supportingText p {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  color: #fff;
+  margin: 24px 24px;
+}
+
+.tags ul {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.tags ul li {
+  text-align: center;
+  border-right: 1px solid #81818c;
+  padding: 0 10px 0 10px;
+}
+
+.tags ul li a p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.tags ul li:last-child {
+  border-right: none;
+}
+
+.buttons {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.buttons button {
+  width: auto;
+}
+
+.containerSkils {
+  display: inline;
+}
+
+.skill {
+  float: left;
+  text-align: center;
+  flex-grow: 1;
+  position: relative;
+  border-right: 1px solid #81818c;
+}
+
+.blurContent {
+  mix-blend-mode: multiply;
+  filter: blur(24px);
+}
+
+.modal {
+  width: 327px;
```

Comment:
[OPTIONAL]
On the mobile version, where there is a big screen, your modal takes a fixed width, when is better to have it responsive.

- [ ] Please fix the width of your pop-up at mobile version with wide screens.
Your design at 990px:
![image](https://user-images.githubusercontent.com/83596135/157799789-bd9d34ff-cb47-4307-893e-20ab31bb27a0.png)